python/labbox_ephys/extractors/bandpass_filter.py

```diff
@@ -63,15 +75,15 @@ def _do_filter(self, chunk):
         #    for m in range(M):
         #        chunk2[m,:]=chunk2[m,:]-np.mean(chunk2[m,:])
         # Do the actual filtering with a DFT with real input
-        chunk_fft = np.fft.rfft(chunk2)
+        chunk_fft = scipy.fft.rfft(chunk2, workers=1)
```

Comment:
Switched from np.fft.rfft to scipy.fft.rfft. Probably doesn't make too much of a difference. But with scipy I could specify the number of workers = 1. That way we don't have as much of a chance of overloading the CPUs if many tasks are happening simultaneously.